src/deployment/android.md

```diff
@@ -159,7 +159,7 @@ that contains a reference to your keystore:
 storePassword=<password from previous step>
 keyPassword=<password from previous step>
 keyAlias=upload
-storeFile=<location of the key store file, such as /Users/<user name>/upload-keystore.jks>
+storeFile=<location of the key store file, such as /Users/<user name>/upload-keystore.jks or C:\\Users\\<user name>\\upload-keystore.jks>
```

Comment:
Hey @loic-sharma can I get you assistance here? I believe the double back-slashes are required here, but can you confirm?

Reply:
@domesticmouse : Within a code block, Markdown requires the double backslash to [escape the backslash](https://daringfireball.net/projects/markdown/syntax#backslash).